preprocessor/src/lib.rs

```diff
@@ -263,14 +266,12 @@ mod tests {
         finalized_sync_committee_branch.insert(0, c.sync_committee_branch[0].clone());
         finalized_sync_committee_branch[1] = c.sync_committee_branch[1].clone();
         c.sync_committee_branch = finalized_sync_committee_branch;
-        // Replaces the attested header with step circuits finalized header
-        c.finalized_header = s.finalized_header.clone();
 
         let params: ParamsKZG<Bn256> = gen_srs(K);
-
+        let pinning = Eth2ConfigPinning::from_path(STEP_CONFIG_PATH);
         let circuit = StepCircuit::<Testnet, Fr>::create_circuit(
             CircuitBuilderStage::Mock,
-            None,
+            Some(pinning),
```

Comment:
you can set pinning to None here, why? `committee_update_testnet.json` and `sync_step_testnet.json` are "prod" configs... if this test override it and we stage it may lead to unexpected proofgen faliures in testnet

Reply:
Good point. I'll keep the pinning but ill change that test case to use a different non prod. config